jvm/src/test/scala/mouse/StringJvmTests.scala

```diff
@@ -12,13 +32,6 @@ class StringJvmTests extends MouseSuite with StringJvmSyntax {
   }
 
   test("parseURL") {
-    implicit val urlEq: Eq[URL] = Eq.fromUniversalEquals
-    implicit val malformedURLExceptionEq: Eq[MalformedURLException] =
-      new Eq[MalformedURLException] {
-        override def eqv(x: MalformedURLException, y: MalformedURLException): Boolean =
-          x.getMessage == y.getMessage
-      }
-
     assertEquals(
```

Comment:
Just curious how this change comes into it.. is `Eq[URL]` supported out of the box now or something?

Reply:
It seems it was unused after migration to MUnit. `assertEquals` isn't integrated with `cats.Eq`.